### ISSUE TITLE
Portal - Updating the left hand side panel of facets 

### ISSUE THREAD

**Issue by gokul472** [overture-stack/dms-ui#2]
## Objective
The objective of this ticket is to ensure that the facets from the demo portal are updated to match the requirement for DMS portal

## Acceptance Criteria
To style the data portal as mocked up, with the functionality already in Arranger

## Zeplin Links for MVP

1. Basic facets: https://zpl.io/2jGGXBr
2. Search facet values (and more values expanded for study facet) https://zpl.io/VDyyrRJ
3. Queried facet panel: https://zpl.io/2ZWWG5l

## Wiki page and for Dev tasks
https://wiki.oicr.on.ca/display/OV/UI+Specs+for+DMS+Portal#UISpecsforDMSPortal-MVPitemsforthePortal

**Comment by b-f-chan** [comment]
Testing blocked waiting on #4 to get default metadata into Arranger, will carryover to Sprint 7

**Comment by b-f-chan** [comment]
Arranger setup complete, testing on QA unblocked. Staging still missing Arranger project, I will try and set that up myself today. But will do initial test on QA as well. Unblocking issue.

**Comment by b-f-chan** [comment]
Styling matches what is provided in Zeplin except for the following:

- Enabled checkbox state has a different colour and style than in the mockups, this is a limitation in Arranger and a workaround or hackaround may be required - Can be investigated in separate ticket

Smoke test on Staging to be done once release tag deployed

*Open question:*

What are the default fields we should be including in the default config?  This is based on the SONG setup but in Arranger a total of 43 fields are available for the facet panel.  Or do we not want to impose a default since we cannot assume what domain the portal is going to be used for (cancer?  weather?  other) and put this entirely in the hands of the configurer?  However, should a reasonable default still not be defined for consistency?

**Comment by rosibaj** [comment]
Lgtm on staging.  closing this ticket as feedback is captured elsewhere.
